libres/lib/enkf/tests/enkf_update_log_test.cpp

```diff
@@ -48,6 +50,7 @@ void test_obs_in_log() {
 }
 
 void test_local_inactive() {
+    std::vector<int> step_list = {10};
```

Comment:
Style: (ignore if you no like)
```suggestion
    std::vector step_list{10};
```
1. `<int>` can be remove because the compiler will deduce the type from the type of the `initalizer_list` you give it (`{10}` in this case).
2. You may delete `=`. This is mostly stylistic as far as I know.